### PR TITLE
Get-GeoFromIpAndTagIncident Playbook Issue fix

### PR DESCRIPTION
**Pull request by v-amolpatil**
Required items, please complete
   
   Change(s):
   - Changes for Get-GeoFromIpAndTagIncident where ip-api was failing because http is required instead of https.

   Reason for Change(s):
   - There is github issue for which we have fixed this. 

   Version Updated:
   - No

   Testing Completed:
   - No

   Checked that the validations are passing and have addressed any issues that are present:
   - Yes

**Testing Images:**
image

image

### CODE REVIEW

File: Playbooks/Get-GeoFromIpAndTagIncident/customConnector/azuredeploy.json [hunk 1]
```diff
@@ -6,29 +6,43 @@
...
-                        "title": "IP-API",
+                        "title": "[parameters('ConnectorName')]",
                         "description": "IP-API Geolocation API",
                         "version": "1.0"
                     },
-                    "host": "ip-api.com",
+                    "host": "[replace(replace(parameters('ServiceEndpoint'),'https://',''),'http://','')]",
                     "basePath": "/json",
                     "schemes": [
                         "https"
```
Comment (rahul0216) [inline Playbooks/Get-GeoFromIpAndTagIncident/customConnector/azuredeploy.json:48]: Schemes should also be http instead of https.

**Review by rahul0216 — COMMENTED** [review]
Please address the inline comments.

File: Playbooks/Get-GeoFromIpAndTagIncident/customConnector/azuredeploy.json [hunk 2]
```diff
@@ -6,29 +6,43 @@
...
     "parameters": {
+        "ConnectorName": {
+            "defaultValue": "IPAPICustomConnector",
+            "type": "String",
+            "metadata": {
+                "description": "Name of the custom IP-API Connector."
+            }
+        },
+        "ServiceEndpoint": {
+            "defaultValue": "https://ip-api.com/json",
```
[the remark this answers is not in the document]
Comment (v-amolpatil) [reply Playbooks/Get-GeoFromIpAndTagIncident/customConnector/azuredeploy.json]: done

File: Playbooks/Get-GeoFromIpAndTagIncident/customConnector/azuredeploy.json (hunk 1 above)
Comment (v-amolpatil) [reply Playbooks/Get-GeoFromIpAndTagIncident/customConnector/azuredeploy.json]: done

**Review by rahul0216 — APPROVED** [review]
Changes looks good. Thanks

**Review by v-rucdu — APPROVED** [review]

**Pull request merged**